Gemfile

```diff
@@ -4,11 +4,3 @@ source "https://rubygems.org"
 ruby "2.4.1"
 
 gemspec path: "."
-gemspec path: "decidim-core"
-gemspec path: "decidim-system"
-gemspec path: "decidim-admin"
-gemspec path: "decidim-dev"
-gemspec path: "decidim-api"
-gemspec path: "decidim-pages"
-gemspec path: "decidim-comments"
-gemspec path: "decidim-meetings"
```

Comment:
No problem! It's a pleasure to contribute to software made with such care for quality and :heart:.